test/indexer/__init__.py

```diff
@@ -137,9 +141,22 @@ def _get_all_hits(self):
                          index=','.join(self.index_service.index_names(self.catalog)),
                          preserve_order=True))
         for hit in hits:
+            entity_type, doc_type = self._parse_index_name(hit)
+            if (
+                entity_type == 'datasets'
+                and doc_type is DocumentType.contribution
+                and 'description' in one(hit['_source']['contents']['datasets'])
+            ):
+                # Sparse dataset contributions contain no lists
```

Comment:
```suggestion
                # DUOS dataset contributions contain no lists
```

there are a few other places where "sparse" is still used.